### ISSUE TITLE
Referring to products

### ISSUE THREAD

**Issue by agentmorris** [sigchi/Document-Formats#90]
It’s often necessary to refer to a product, as in:

Software analysis was done in Matlab (MathWorks, Inc.).

…or:

For this study, participants wore an Apple Watch (Apple, Inc.).

There have been two unofficial conventions for doing this: parentheticals and footnotes, and then like 5% of the time someone does this in a reference.  It would be good to clarify where to put this information, and what to include.  I specifically vote for: (1) parentheticals, and (2) don’t list the city where the company’s headquarters are, that seems really dated.  But mostly I vote for clarity.  And for _not_ doing this in references.  I’m 60% for parentheticals, 40% for footnotes, 0% for references.

**Comment by ayman** [comment]
I'd agree mostly.  Not sure if we should just write it in or if we should ask for a broader community vote.  First, I'd say we figure out what other style guides say.

**Comment by agentmorris** [comment]
I searched briefly and neither APA nor MLA appear to have clear guidelines here; they discuss citing product instruction manuals, but that's a different thing.

I say we invite folks who care to comment on this thread, and unless there is wild disagreement, just do it; this is too small an issue to make a big discussion out of IMO.

I'll post a link to this thread publicly.

-Dan

**Comment by danielavrahami** [comment]
My vote:
60% footnote
40% references
0% parentheses

Recently, my vote counts for s#$t

**Comment by ayman** [comment]
I think the argument is to declutter references.  This assumes it's not important to aggregate all the references to apple.com across the dl...course with out a proper DOI for apple.com (for example) it's no different than a `fgrep apple.com *` - so I'd argue for footnoting.  @toupsz recently came up with a video game citation (which is a bit different but related)...maybe he can comment too.

**Comment by danyelf** [comment]
I tend to find myself choosing none vs footnote vs parens based on length, and context (learning that an Apple Watch is made by Apple Inc isn't very interesting to me).

we uploaded relevant content from the Wikipedia Download Center \1 to our Apple Watch using TransCopyLink (CopyProducts Inc.)

\1 As of Nov 1, 2016: https://dumps.wikimedia.org/enwiki/20161101/

**Comment by snapdragoness** [comment]
I've generally used a footnote to indicate the model and/or URL for products. Software can be cited (much like the game citation method, which I got from the Game Studies journal, but I think is just an adaptation of software); there's a model for it somewhere. I have seen software cited especially for analysis, since you want to indicate what version of the software was used (since there could be some error or set of design decisions that went into the analysis algorithm). 

@danyelf Knowing that Apple made the Apple Watch may not be too exciting, but model and screen size are relevant...

**Comment by snapdragoness** [comment]
This was a page I encountered when looking into citing software for analysis some months back:

http://www-01.ibm.com/support/docview.wss?uid=swg21476197

**Comment by ayman** [comment]
I'm wondering if we can have a rule of thumb that says something like: _**if it doesn't need a h-index score, leave it out of the references**_. If the effect is to remove the dot coms from the references but the consequence is I get refs for free these days, parens and footnotes eat into my page length.

**Comment by agentmorris** [comment]
Ayman's point about impact on page length is a good one, and - equally pragmatically - footnotes can be a little flakier to work with and precisely control than references in both Word and LaTeX.  Thinking of it this way makes me change my ordering to 60% parentheticals, 40% references, 0% footnotes.  Or more specifically, if we decide that you should cite a company name when referring to a product, a parenthetical (without company location) seems appropriate, as in my original examples:

Software analysis was done in Matlab (MathWorks, Inc.).
For this study, participants wore an Apple Watch (Apple, Inc.).

In both these cases, a URL would be silly.  Then if there's a case where you need to refer to a specific URL for some reason, I would now argue for a reference (rather than a footnote) in that case.

This suggestion still leaves open the question of whether a company name is necessary at all; I would say yes, largely to highlight that something _is_ a product, which maybe is obvious in the case of the Apple Watch, but is often not so clear.  Also in terms of longevity, products by construct are shorter-lived than the companies that make them on average.

-Dan

**Comment by ayman** [comment]
I think @agentmorris hits upon the complexity of product.  I can cite an artifact.  The Apple Watch v1 vs v2 becomes similar to @toupsz idea about video games...there's a product.  Who's to say the Facebook that user #284 saw in their i18n, l10n, and bucket test on Sept 25, 2016.  There's no real version or anything to cite for a baseline.  So if we could think of things with exact instances (hardware with hard releases) or software circa a date/time stamp...it becomes like a URL reference and nothing more than to say this was on the Wikipedia accessed Sept 25, 2016.

**Comment by danielavrahami** [comment]
The more I think about the topic, and with the help of all of your comments, I become more convinced that there is no good justification at the moment to try and standardize this. For one, it appears that there are many categories of references to products and companies that a solution would need to be either a blanket solution that is inappropriate for some cases, or be so specific that it is hard to follow.  And furthermore, making such decisions are often hard to change and can strand the community with sub-optimal solutions (as an example, consider that the CHI references still use "Firstname Lastname" instead of "Lastname Firstname", despite it being undesired by most. )

**Comment by ayman** [comment]
> making such decisions are often hard to change and can strand the community with sub-optimal solutions

That was the ACM format which we conformed to...the danger here is there is no higher format. Which is why I'm against outlining a practice, but the wiki could highlight a few ways to do this.  Not to endorse your opinion more @danielavrahami - but my thought now is we close this and if @agentmorris want's to highlight some 'best practices' in the wiki then that's fine.

**Comment by agentmorris** [comment]
All OK by me.  I love the idea of standardizing everything, but agree that this thread is enough to convince me there's not a clear answer.
